tests/unit/test_query_language.py

```diff
@@ -63,3 +74,50 @@ def test_dataset() -> None:
             rhs=Value(2000),
         ),
     }
+
+
+# The problem: We'd like to test that operations on query language (QL) elements return
+# the correct query model (QM) elements. We like tests that emphasise what is being
+# tested, and de-emphasise the scaffolding. We dislike test code that looks like
+# production code.
```

Comment:
I don't understand what "we dislike test code that looks like production code" means.

Reply:
This was my summary of your concern that chunks of the test code could look like a copy-and-paste from the production code. For example, test code:

```python
class TestIntSeries:
    def test_le_value(self):
        assert_produces(
            IntSeries(qm_int_series) <= 2000,
            Function.LE(qm_int_series, Value(2000)),  # This
        )
```

Production code:

```python
class IntSeries(Series):
    def __le__(self, other):
        return BoolSeries(Function.LE(lhs=self.qm_node, rhs=Value(other)))  # Looks like this
```

In earlier iterations, the resemblance was far more apparent.